### PR TITLE
Updated ReadMe to include requirements

### PR DESCRIPTION
**Pull request by jamesqquick**
Included requirement to install LaTeX and where to install it.

### CODE REVIEW

File: readme.md [hunk 1]
```diff
@@ -1,6 +1,10 @@
 # node-latex
 A small wrapper for generating PDFs with LaTeX in Node.
 
+## Requirements
+
+LaTeX must be installed on your machine.  You can download [here](https://www.latex-project.org/get/).
```
Comment (saadq) [inline readme.md]: Can you just change `You can download here` to `You can download it here`

**Comment by saadq** [comment]
Yeah, I suppose this would be a good message to include. Just one small correction that I mentioned above if you don't mind.

**Comment by jamesqquick** [comment]
Updated wording on requirements.

**Comment by saadq** [comment]
Thanks very much for the contribution! :)

**Pull request merged**